test/e2e/tests/network/update-network.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { strict as assert } from 'assert';
+import { WebElement } from 'selenium-webdriver';
+import { Suite } from 'mocha';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+const selectors = {
+  accountOptionsMenuButton: '[data-testid="account-options-menu-button"]',
+  informationSymbol: '[data-testid="info-tooltip"]',
+  inputText: 'input[type="text"]',
+  settingsOption: { text: 'Settings', tag: 'div' },
+  networkOption: { text: 'Networks', tag: 'div' },
+  ethereumNetwork: { text: 'Ethereum Mainnet', tag: 'div' },
+  deleteButton: { text: 'Delete', tag: 'button' },
+  cancelButton: { text: 'Cancel', tag: 'button' },
+  saveButton: { text: 'Save', tag: 'button' },
+  updatedNetworkDropDown: { tag: 'span', text: 'Update Network' },
+  errorMessageInvalidUrl: {
+    tag: 'h6',
+    text: 'URLs require the appropriate HTTP/HTTPS prefix.',
+  },
+};
+
+async function editNetworkDetails(
+  driver: Driver,
+  indexOfInputField: number,
+  inputValue: string,
+) {
+  const getAllInputElements: WebElement[] = (await driver.findElements(
+    selectors.inputText,
+  )) as WebElement[];
+  const inputTextFieldToEdit: WebElement =
```

Comment:
Can we select the input text field based on a selector specific to the input? If we are able to select it based on placeholder text or a data-test-id, instead of an index number, then this test will not break if new input fields are added.